setup.py

```diff
@@ -42,17 +42,13 @@
                 'geopandas>=0.8.0',
                 'descartes>=1.0.0',
                 'shapely>=1.6.0',
-                'scikit-learn>=0.23.0',
-                'shap>=0.36.0',
                 'pandas>=0.23.0',],
         'plot': ['graphviz>=0.9.0', 
                  'python-highcharts>=0.4.1',],
         'geo': ['geopandas>=0.8.0',
                 'descartes>=1.0.0',
                 'shapely>=1.6.0'],
         'ml': ['tqdm>=4.0.0',
-               'scikit-learn>=0.23.0',
-               'shap>=0.36.0',
```

Comment:
Why do you remove these from the list of requirements?

Reply:
we do not need them anymore. SHAP and to_sklearn are deprecated. to_sklearn is replaced by to_memmodel and to_shapExplainer will be replaced by our own implementation.